xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/WebHome.xml

```diff
@@ -47,23 +47,28 @@
     [[image:icon:wand]] $services.localization.render('xe.panels.customize') [[$services.localization.render('panelwizard.panelwizard')&gt;&gt;Panels.PanelWizard]].
   {{/box}}
 #end
-#set($columns = ["name", "description", "type", "category", '_actions'])
-#set($columnsProperties = {
-  "name" : { "type" : "text", "link" : "view" },
-  "description" : { "type" : "text", "html" : "true" },
-  "type" : { "type" : "list" },
-  "category" : { "type" : "list" },
-  '_actions': {'html': true, 'sortable': false, 'actions': ['edit', 'delete']}
+#set ($liveDataConfig = {
+  'meta': {
+    'propertyDescriptors': [
+      { 'id': 'name', 'displayer': 'link' }
+    ]
+  }
 })
-#set($options = { 
-   "className":"Panels.PanelClass",
-   "selectedColumn":"name",
-   "defaultOrder":"asc",
-   "translationPrefix" : "panels.",
-   "rowCount": 30,
-   "queryFilters": "currentlanguage"
-})
-#livetable("panels" $columns $columnsProperties $options)
+#set ($sourceParameters = $escapetool.url({
+  'className': 'Panels.PanelClass',
+  'selectedColumn': 'name',
+  'defaultOrder': 'asc',
+  'translationPrefix' : 'panels.',
+  'rowCount': 30,
```

Comment:
Live Data has **its own way** to configure the pagination. You should use that instead, i.e. the ``limit`` macro parameter. See https://extensions.xwiki.org/xwiki/bin/view/Extension/Live%20Data%20Macro/#HParameters .

As a general remark, the only source parameters that you should pass to the live table source (if needed) are listed here https://extensions.xwiki.org/xwiki/bin/view/Extension/Live%20Data%20Macro/#HParameters-1 .